clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -216,5 +219,11 @@ type schedulingMetrics struct {
 	PriorityEvaluationLatency   measurementutil.LatencyMetric `json:"priorityEvaluationLatency"`
 	PreemptionEvaluationLatency measurementutil.LatencyMetric `json:"preemptionEvaluationLatency"`
 	BindingLatency              measurementutil.LatencyMetric `json:"bindingLatency"`
-	E2eSchedulingLatency        measurementutil.LatencyMetric `json:"e2eSchedulingLatency"`
+	E2eSchedulingLatency        measurementutil.HistogramVec  `json:"e2eSchedulingLatency"`
```

Comment:
I think a cleaner design is to make the E2eSchedulingLatency conform to the LatencyMetric struct (you may want to add Average to the existing LatencyMetric since average is important too). And you shouldn't need any changes on the perfdash side. 

BTW, this [PR](https://github.com/kubernetes/kubernetes/pull/85861) has logic on how to approximately calculate percentiles from Histograms. 
If it's too complex, feel free to just implement average latency without percentiles.

Reply:
Got it,  I misunderstood it.